src/libYARP_OS/src/Companion.cpp

```diff
@@ -1563,63 +1563,24 @@ int Companion::cmdClean(int argc, char *argv[]) {
 int Companion::cmdResource(int argc, char *argv[]) {
     if (argc==0) {
         printf("Looks for, and prints the complete path to, resource files.\n");
-        printf("Example usage (from RobotCub project):\n");
-        printf("   yarp resource --find icub.ini\n");
-        printf("   yarp resource --find config --dir\n");
-        printf("   yarp resource --icub icub.ini --find icub\n");
-        printf("   yarp resource --from config.ini --find icub\n");
-        printf("   yarp resource --find icub.ini --verbose 1\n");
-        printf("   yarp resource --find config.ini --all\n");
+        printf("Example usage:\n");
+        printf("   yarp resource --context context-name --from file-name\n");
+        printf("   yarp resource --context context-name --find file-name\n");
```

Comment:
Perhaps it would be useful to mention in the help that the `YARP_ROBOT_NAME` variable also influences the result